tests/vppoutputencode.cpp

```diff
@@ -160,13 +162,37 @@ bool VppOutputEncode::config(NativeDisplay& nativeDisplay, const EncodeParams* e
     m_encoder->setNativeDisplay(&nativeDisplay);
     m_mime = m_output->getMimeType();
     setEncodeParam(m_encoder, m_width, m_height, encParam, m_mime);
-
+    printLatency = encParam->printFrameLatency;
     Encode_Status status = m_encoder->start();
     assert(status == ENCODE_SUCCESS);
     initOuputBuffer();
     return true;
 }
 
+void VppOutputEncode::getFrameDelay(VideoEncOutputBuffer m_outputBuffer){
+    struct timeval end;
+    gettimeofday(&end,NULL);
+    int64_t now = 1000000 * end.tv_sec + end.tv_usec;
+    int64_t delay = now - m_outputBuffer.timeStamp;
+    m_FrameCount++;
+    if (m_FrameCount == 1){
+        m_MaxLatency = delay;
+        m_MinLatency = delay;
+        m_AveLatency = delay;
+    }
+    else{
+        if (delay > m_MaxLatency) 
+            m_MaxLatency = delay;
+        if (delay < m_MinLatency)
+            m_MinLatency = delay;
+        m_AveLatency = (m_AveLatency * (m_FrameCount - 1) + delay) / m_FrameCount;
```

Comment:
Please do not use m_AveLatency, just sum it up, the latency is small. it will not overflow